kura/org.eclipse.kura.core.net/src/main/java/org/eclipse/kura/core/net/FirewallConfiguration.java

```diff
@@ -68,8 +68,8 @@ public FirewallConfiguration(Map<String, Object> properties) {
     private void parseNatRules(Map<String, Object> properties) {
         String str;
         String[] astr;
```

Comment:
I would suggest to use more meaningful names for some variables.

`str` -> `openPorts`
`astr` -> `rules`
`private void parseOpenPortRule(String sop)` -> `private void parseOpenPortRule(String **rule**)`